core/src/cluster_info.rs

```diff
@@ -115,7 +115,7 @@ pub const DEFAULT_CONTACT_SAVE_INTERVAL_MILLIS: u64 = 60_000;
 /// Minimum serialized size of a Protocol::PullResponse packet.
 const PULL_RESPONSE_MIN_SERIALIZED_SIZE: usize = 161;
 // Limit number of unique pubkeys in the crds table.
-const CRDS_UNIQUE_PUBKEY_CAPACITY: usize = 4096;
+pub(crate) const CRDS_UNIQUE_PUBKEY_CAPACITY: usize = 4096;
```

Comment:
Could we handle 8k?

Reply:
This was mentioned in: https://github.com/solana-labs/solana/pull/15539#discussion_r590519612
It will have 830x impact on size of the crds table. So I am worried that would be too much. For now this is 4x tds cluster and 6x mainnet.

Reply:
Ok.